### PR TITLE
fix transformer version

### PR DESCRIPTION
**Pull request by ivy-lv11**
Specify transformers version to 4.36.0

### CODE REVIEW

**Review by Oscilloscope98 — COMMENTED** [review]
Let's uninstall sentence-transformers after the llamaindex ut finished. And put the unsinatll task instead a always true step :)

File: .github/workflows/llm_unit_tests.yml [hunk 1]
```diff
@@ -223,9 +223,14 @@ jobs:
...
           pip install llama-index-readers-file llama-index-vector-stores-postgres llama-index-embeddings-huggingface
-          pip install transformers==4.31.0
+          pip install transformers==4.36.0
           pip install "pydantic>=2.0.0"
           bash python/llm/test/run-llm-llamaindex-tests.sh
+      - name: Run sentence-transformers uninstallation
+        if: ${{ always() }}
+        shell: bash
+        run: |
+          pip uninstall sentence-transformers
```
Comment (Oscilloscope98) [inline .github/workflows/llm_unit_tests.yml:233]: pip uninstall sentence-transformers -y || true

File: .github/workflows/llm_unit_tests.yml [hunk 2]
```diff
@@ -399,5 +404,11 @@ jobs:
...
+          pip install transformers==4.36.0
           pip install "pydantic>=2.0.0"
-          bash python/llm/test/run-llm-llamaindex-tests-gpu.sh
\ No newline at end of file
+          bash python/llm/test/run-llm-llamaindex-tests-gpu.sh
+      - name: Run sentence-transformers uninstallation
+        if: ${{ always() }}
+        shell: bash
+        run: |
+          pip uninstall sentence-transformers
```
Comment (Oscilloscope98) [inline .github/workflows/llm_unit_tests.yml:414]: same here

File: .github/workflows/llm_unit_tests.yml (hunk 1 above)
Comment (ivy-lv11) [reply .github/workflows/llm_unit_tests.yml]: Added

File: .github/workflows/llm_unit_tests.yml (hunk 2 above)
Comment (ivy-lv11) [reply .github/workflows/llm_unit_tests.yml]: Added

**Review by Oscilloscope98 — APPROVED** [review]

**Pull request merged**